receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_metadata.go

```diff
@@ -21,8 +21,11 @@ import (
 // labels
 
 const (
-	deviceLabelName = "device"
-	stateLabelName  = "state"
+	deviceLabelName         = "device"
+	filesystemTypeLabelName = "fs.type"
```

Comment:
Not sure you need the `fs.`. The metrics here are already scoped to `...filesystem...` so I think that is redundant?

Reply:
True, but I believe there's value in adding the `fs.` prefix to avoid label key collision since `type` is pretty generic. FWIW other labels in the convention also follow this pattern where there's a prefix indicating what entity the label describes. What do you think?

Reply:
Hmm not sure, what are some examples of where we have labels with a prefix?

I was thinking of the `state` label that has a different meaning across a bunch of different metrics (CPU, memory, , etc) which we don't prefix.

Reply:
I was thinking about the conventions [here](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions) - label names themselves give some information about the entity to which they are associated with. But I see, those are resource labels whereas these are metric labels. Will update `fs.type` to `type` to make it consistent with the rest of the metric labels.